ci/build/build-ray-docker.sh

```diff
@@ -6,11 +6,12 @@ SOURCE_IMAGE="$2"
 CONSTRAINTS_FILE="$3"
 DEST_IMAGE="$4"
 PIP_FREEZE_FILE="$5"
+DOCKERFILE="$6"
 
 CPU_TMP="$(mktemp -d)"
 
 cp -r .whl "${CPU_TMP}/.whl"
-cp docker/ray/Dockerfile "${CPU_TMP}/Dockerfile"
+cp "${DOCKERFILE}" "${CPU_TMP}/Dockerfile"
```

Comment:
as discussed, please revert these change. rather, please add an additional layer on top of ray image to install podman.